pkg/ccl/changefeedccl/schemafeed/schema_feed.go

```diff
@@ -480,36 +480,37 @@ func (tf *schemaFeed) pauseOrResumePolling(ctx context.Context, atOrBefore hlc.T
 	// Always assume we need to resume polling until we've proven otherwise.
 	tf.mu.pollingPaused = false
 
-	if err := tf.targets.EachTableID(func(id descpb.ID) error {
+	if canPausePolling, err := tf.targets.EachTableIDWithBool(func(id descpb.ID) (bool, error) {
 		// Check if target table is schema-locked at the current frontier.
 		ld1, err := tf.leaseMgr.Acquire(ctx, frontier, id)
 		if err != nil {
-			return err
+			return false, err
 		}
 		defer ld1.Release(ctx)
 		desc1 := ld1.Underlying().(catalog.TableDescriptor)
 		if !desc1.IsSchemaLocked() {
-			return errors.Newf("desc %d not schema-locked at frontier %s", desc1.GetID(), frontier)
+			log.VInfof(ctx, 2, "desc %d not schema-locked at frontier %s", desc1.GetID(), frontier)
```

Comment:
Would have to look at the benchmark to be sure, but my guess is that putting this behind `if log.V(2)` would help in the common case. But, it probably doesn't matter too much as you've already improved this function substantially.

Reply:
I know I am the one who recommended this, but I wonder if there is really a world in production where we'd want to print this on every message. I know we were doing it before implicitly by logging the error at V(1) so I leave it to y'alls judgement.

Reply:
> my guess is that putting this behind if log.V(2)

Using V(1) sounds good to me which matches what we do for other error cases https://github.com/cockroachdb/cockroach/blob/f701074bdb42dbd7e078b3ce8a0ee4cbaca82fab/pkg/ccl/changefeedccl/schemafeed/schema_feed.go#L527-L529.